### ISSUE TITLE
Continue Training from a saved checkpoint

### ISSUE THREAD

**Issue by Jasper-Duong** [megvii-research/NAFNet#79]
Dear honorable authors,
In the config files for training process, there is `path` for `pretrain_network_g` and `resume_state`. I wonder if they are used to load model at a certain checkpoint or a pretrained model in order to resume training from there? Since I usually train with just 2 GPUs, sometimes 1, with limited time allowed for a single use in my college lab, training from a saved checkpoint seems to be a suitable way to learn more about your amazing work (simple yet effective methods).

Thank you for providing such wonderful resources for the image restoration task and I look forward to hearing your response

**Comment by mayorx** [comment]
Hi, Jasper-Duong,
Thanks for your attention to NAFNet.
Yes,  set ``pretrain_network_g`` to the path of the checkpoint and starting training from the pre-trained model is a common practice. Sometimes, the training settings may need to be modified, e.g. a reduced learning rate to prevent the destruction of the pre-trained weights.
